src/libsyntax/ast.rs

```diff
@@ -2605,13 +2605,13 @@ pub type ForeignItem = Item<ForeignItemKind>;
 /// An item within an `extern` block.
 #[derive(Clone, RustcEncodable, RustcDecodable, Debug)]
 pub enum ForeignItemKind {
-    /// A foreign function.
-    Fn(FnSig, Generics, Option<P<Block>>),
-    /// A foreign static item (`static ext: u8`).
+    /// A static item (`static FOO: u8`).
     Static(P<Ty>, Mutability),
-    /// A foreign type.
+    /// A function.
+    Fn(FnSig, Generics, Option<P<Block>>),
+    /// A type.
     Ty,
-    /// A macro invocation.
+    /// A macro expanding to an item.
```

Comment:
```suggestion
    /// A macro expanding to items.
```
`0` or more of them.
(This is applicable to other macro items.)